package-testing/testing-api/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:18
+
+WORKDIR /app
+
+COPY package*.json ./
```

Comment:
Since the source is likely to change more often than the imports, copy package.json first, install, then copy the src. Makes better use of Docker layers than just copying everything then running `yarn install` and `yarn build`